src/pages/Exhibitions/Section/Section.tsx

```diff
@@ -0,0 +1,15 @@
+// Libraries
+import React from "react";
+import SectionProps from "./models/SectionProps";
+// Styles
+import "./style.css";
+
+const Section: React.FC<SectionProps> = (props: SectionProps) => (
+  <div>
+    {/* <h2>{props.sectionName}</h2>
+    <br />
+    <h3>{props.exhibitor}</h3> */}
```

Comment:
@sophiewongsp Same as the comment above for `Role.tsx`. You can refer to [`ProjectRowText`](https://github.com/niji-co/Fusion-Web/blob/master/src/pages/Project/Row/ProjectRowText.tsx) for how I did it. (There are other ways too so feel free to use them too)